actionlist/app/reducers/action_items.js

```diff
@@ -8,9 +8,24 @@ export const defaultState = {
   drawerExpanded: false,
 };
 
+// ToggleHelper to toggleback already opened card when other card is clicked
+const toggleHelper = (dataSource) => {
+  const temp = {};
+  Object.keys(dataSource).forEach((key) => {
```

Comment:
A `forEach` will modify your object in place. You should be using `.map` or some other function which returns a new object/array when you want to update things in state.